pkg/reconciler/v1alpha1/pipelinerun/pipelinerun.go

```diff
@@ -441,20 +442,20 @@ func (c *Reconciler) updateTaskRunsStatusDirectly(pr *v1alpha1.PipelineRun) erro
 func (c *Reconciler) createTaskRun(logger *zap.SugaredLogger, rprt *resources.ResolvedPipelineRunTask, pr *v1alpha1.PipelineRun, storageBasePath string) (*v1alpha1.TaskRun, error) {
 	var taskRunTimeout = &metav1.Duration{Duration: 0 * time.Second}
```

Comment:
I wonder if 0 is a bad default here, if that means no timeout? And/or maybe we should use `apisconfig.NoTimeoutDuration` directly if we intend for that to be the default value of this var?